### DISCUSSION TITLE
v1.4.3

### DISCUSSION THREAD

**Discussion by Birdulon** [Melledy/Grasscutter#1936] [Announcements]
## What's Changed
* Updated spanish translations by @RBNXI in https://github.com/Grasscutters/Grasscutter/pull/1911
* [Security][Bugfix] Fix directory traversal exploit by @sandtechnology in https://github.com/Grasscutters/Grasscutter/pull/1907
* Debug system by @BiosNod in https://github.com/Grasscutters/Grasscutter/pull/1894
* Fix Mail SendToAll Freeze by @jie65535 in https://github.com/Grasscutters/Grasscutter/pull/1913
* Update README_ja-JP.md by @Ign1s-Reiga in https://github.com/Grasscutters/Grasscutter/pull/1922
* Add the new keys (3.2 Support) by @lilmayofuksu in https://github.com/Grasscutters/Grasscutter/pull/1923
* fix: causes NPE when logging in because roomSceneId is assigned incorrectly. by @hamusuke0323 in https://github.com/Grasscutters/Grasscutter/pull/1926
* Various bugfixes and refactoring

## New Contributors
* @sandtechnology made their first contribution in https://github.com/Grasscutters/Grasscutter/pull/1907
* @BiosNod made their first contribution in https://github.com/Grasscutters/Grasscutter/pull/1894
* @Ign1s-Reiga made their first contribution in https://github.com/Grasscutters/Grasscutter/pull/1922

**Full Changelog**: https://github.com/Grasscutters/Grasscutter/compare/v1.4.2...v1.4.3

<hr /><em>This discussion was created from the release <a href='https://github.com/Grasscutters/Grasscutter/releases/tag/v1.4.3'>v1.4.3</a>.</em>

**Comment by nitrog0d** [comment]
![IMG_5249.jpg](https://user-images.githubusercontent.com/31241699/200201681-497383a9-1934-433a-abc5-ed26e4ee9826.jpg)

**Comment by jia-dh** [comment]
Hello,developer.I want to ask a question. When to update files？🙂

**Reply by nitrog0d** [reply]
![20221218_120203.jpg](https://user-images.githubusercontent.com/31241699/208956751-3f8cdadd-bee2-4797-bd59-de13857cf2df.jpg)

**Reply by ArthDev1** [reply]
Me lon
